man/org.freedesktop.systemd1.xml

```diff
@@ -12009,6 +12225,15 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
       <para><varname>WantsMountsFor</varname> was added in version 256.</para>
       <para><varname>DebugInvocation</varname>, and
       <varname>CanLiveMount</varname> were added in version 257.</para>
+      <para><varname>StateDirectoryQuota</varname>,
+      <varname>StateDirectoryAccounting</varname>,
+      <varname>StateDirectoryQuotaUsage</varname>,
+      <varname>CacheDirectoryQuota</varname>,
+      <varname>CacheDirectoryAccounting</varname>,
+      <varname>CacheDirectoryQuotaUsage</varname>,
+      <varname>LogsDirectoryQuota</varname>,
+      <varname>LogsDirectoryAccounting</varname>, and
+      <varname>LogsDirectoryQuotaUsage</varname>, were added in version 258.</para>
```

Comment:
The new properties are available only on service, socket, mount, and swap units.
Here is for all kinds of unit. Please drop.